tox.ini

```diff
@@ -70,10 +70,3 @@ commands =
       {posargs}
 allowlist_externals = rm
 skipsdist = true
-
-[flake8]
-show-source = True
-exclude = venv,.git,.tox,dist,*egg,.env,hack
```

Comment:
`exclude` session is omitted in flake8 section of the toml file above, is it defined elsewhere?

Reply:
I stated that in the commit message, we run flake8 on specific targets so we don't have to exclude venv etc.

Reply:
Missed that one, sorry!